src/ReleaseHistory.md

```diff
@@ -1,6 +1,6 @@
 # SARIF Package Release History (SDK, Driver, Converters, and Multitool)
 
-## UNRELEASED
+## **v2.4.14** [Sdk](https://www.nuget.org/packages/Sarif.Sdk/2.4.14) | [Driver](https://www.nuget.org/packages/Sarif.Driver/2.4.14) | [Converters](https://www.nuget.org/packages/Sarif.Converters/2.4.14) | [Multitool](https://www.nuget.org/packages/Sarif.Multitool/2.4.14) | [Multitool Library](https://www.nuget.org/packages/Sarif.Multitool.Library/2.4.14)
```

Comment:
>. [](http://example.com/codeflow?start=9&length=1)

What's prompting us to release at this specific point in time?

Reply:
We are updating the sarif website with the latest package. If we use the Location Id change, we would be able to parse SARIF files that are using bigInteger values.

Also, we closed this: [https://github.com/microsoft/sarif-sdk/issues/2397](https://github.com/microsoft/sarif-sdk/issues/2397) but we never shipped the actual fix.